toxcore/tox.c

```diff
@@ -1072,6 +1072,16 @@ void tox_kill(Tox *tox)
     mem_delete(tox->sys.mem, tox);
 }
 
+void tox_get_options(Tox *tox, struct Tox_Options *options)
```

Comment:
what is this doing in this pr? Is this some left over?

Reply:
I extracted the code here https://gist.github.com/Green-Sky/60199323907fdc3862196da938abdedf